node/actors/executor/src/lib.rs

```diff
@@ -57,7 +53,7 @@ pub struct Config {
     pub gossip_static_outbound: HashMap<node::PublicKey, net::Host>,
     /// Http debug page configuration.
     /// If None, debug page is disabled
-    pub debug_page: Option<DebugPageConfig>,
+    pub debug_page_config: Option<DebugPageConfig>,
```

Comment:
I'm not sure if I didn't say the opposite in the previous review round (sry, if that's been the case), but "_config" here is redundant, since it is a field of the Config struct.